src/query/storages/common/cache/src/providers/disk_cache.rs

```diff
@@ -122,7 +123,7 @@ where C: Cache<String, u64, DefaultHashBuilder, FileSize>
         self.root.join(rel_path)
     }
 
-    fn init(self) -> self::result::Result<Self> {
+    fn reset_restart(self) -> result::Result<Self> {
```

Comment:
The reset also can be parallelism?

Reply:
It seems OpenDAL has remove_via batch remove: https://github.com/apache/opendal/blob/08b2a96c338815208ea9b7e852034f76965ac8e6/core/src/types/operator/operator.rs#L1318

Reply:
Absolutely, we can make that